cypress.config.js

```diff
@@ -2,7 +2,8 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
-    setupNodeEvents(on, config) {
-    },
-  },
-});
+    baseUrl: 'https://demoqa.com',
+    viewportHeight: 1600,
+    viewportWidth: 1200,
+  }
+});
```

Comment:
```suggestion
});

```

Reply:
> check your code style. look at the [example]  
  wow, it's really familiar code with very familiar credentials)